### ISSUE TITLE
Simple interface of functions such as is_included_antichain 

### ISSUE THREAD

**Issue by kilohsakul** [VeriFIT/mata#113]
Currently they have 5 parameters, three are not used.
Maybe make them optional?
The alphabet parameter should probably dissappear from inclusion and equivalence, regardless nfa having it as a property or not.
If there are more skeletons like this, we chould clean them.

Another issue related to user friendliness of these functions, I am still not sure wheter making more header files and more namespaces for these functions was a good idea. For me it gets hard to call these functions, I need to search for the right name space, include, use strange keyboard shortcuts in CLion, ....  it is stressful for an old person. I can do it eventually, but it eats brain cycles. Not sure what is better here.

**Comment by tfiedor** [comment]
Just for the record. Lukas is talking about following parameters:
```
bool Mata::Nfa::Algorithms::is_included_antichains(
    const Nfa&             smaller,
    const Nfa&             bigger,
    const Alphabet* const  alphabet,
    Run*                   cex,
    const StringMap&      params)
{ // {{{
```
Which are not optional? I believe `params` should be.

**Comment by tfiedor** [comment]
I agree, that both pointers should by default be `nullptr`, and params should provide some "sane default" or be removed alltogether. Why is it in the function? Shouldn't the params be only in the master function that calls particular `is_included_*` functions?

**Comment by martinhruska** [comment]
I've checked this. It looks that parameters are not used yet. The pointer parameters are not big issue since we can use default parameter for them and don't bother a user. We would need to convert `StringMap&` also to pointer or use a default value. It is there since also the antichain based inclusion can have internal paramaters such as a preorder relation.

**Comment by Adda0** [comment]
There is a similar problem with equivalence check (or so says Lukáš).

It was decided that we will provide default values for those parameters, which are necessary, and remove the unused parameters altogether.

**Comment by Adda0** [comment]
@vhavlena Keep this issue in mind when refactoring inclusion algorithms. If you manage to resolve this issue as a by-product of the refactorization, even better.
